server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/navbar.ftl

```diff
@@ -18,73 +18,71 @@
-              </a>
-              <ul class="dropdown-menu">
-                <li><a href="/manager"><span id="managerStatusNotification" class="icon-dot normal"></span>&nbsp;Manager&nbsp;Server&nbsp;</a></li>
-                <li><a href="/tservers"><span id="serverStatusNotification" class="icon-dot normal"></span>&nbsp;Tablet&nbsp;Servers&nbsp;</a></li>
-                <li><a href="/gc"><span id="gcStatusNotification" class="icon-dot normal"></span>&nbsp;Garbage&nbsp;collector&nbsp;</a></li>
-              </ul>
-            </li>
-            <li><a href="/tables">Tables</a></li>
-            <li class="dropdown">
-              <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
-                Activity <span class="caret"></span>
-              </a>
-              <ul class="dropdown-menu">
-                <li><a href="/compactions">Active&nbsp;Compactions</a></li>
-                <li><a href="/scans">Active&nbsp;Scans</a></li>
-                <li><a href="/bulkImports">Bulk&nbsp;Imports</a></li>
-                <li><a href="/ec">External&nbsp;Compactions</a></li>
-                <li><a href="/replication">Replication</a></li>
-              </ul>
-            </li>
-            <li class="dropdown">
-              <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Debug&nbsp;<span id="errorsNotification" class="badge"></span><span class="caret"></span>
-              </a>
-              <ul class="dropdown-menu">
-                <li><a href="/log">Recent&nbsp;Logs&nbsp;<span id="recentLogsNotifications" class="badge"></span></a></li>
-                <li><a href="/problems">Table&nbsp;Problems&nbsp;<span id="tableProblemsNotifications" class="badge"></span></a></li>
-              </ul>
-            </li>
-            <li class="dropdown">
-              <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
-                REST <span class="caret"></span>
-              </a>
-              <ul class="dropdown-menu">
-                <li><a href="/rest/xml">XML Summary</a></li>
-                <li><a href="/rest/json">JSON Summary</a></li>
-              </ul>
-            </li>
-            <li class="dropdown">
-              <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
-                <span class="glyphicon glyphicon-option-vertical"></span>
-              </a>
-              <ul class="dropdown-menu">
-                <li><a class="auto-refresh" style="cursor:pointer">Auto-Refresh</a></li>
-                <li><a data-toggle="modal" href="#aboutModal">About</a></li>
-              </ul>
-            </li>
-          </ul>
-        </div>
-      </div>
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```

Comment:
Not sure what happened, but it looks like the formatting on this whole file got messed up somehow- should probably be returned to proper formatting.